sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -40,10 +42,13 @@ class GestureArena {
 
   void _resolve(Object key, GestureArenaMember member, GestureDisposition disposition) {
     List<GestureArenaMember> members = _arenas[key];
+    if (members == null && disposition == GestureDisposition.accepted)
```

Comment:
You should document, in GestureArenaEntry.resolve(), that it's ok to resolve accepted when you're already accepted, but not rejected when already rejected, and not accepted or rejected when you're already rejected or accepted respectively.